packages/terra-application-layout/src/menu/_ApplicationMenu.jsx

```diff
@@ -81,6 +82,19 @@ class ApplicationMenu extends React.Component {
     }
   }
 
+  handleBlur(event) {
+    const { utilityConfig, layoutConfig } = this.props;
+    const currentTarget = event.currentTarget;
+
+    if (!utilityConfig.onBlur) return;
```

Comment:
You can just update you logic so that the blur event is not registered unless this method is defined.  There does not seem to be a need for this to execute otherwise.